### DISCUSSION TITLE
protobuf-c make failed with: Makefile:2297: recipe for target 't/test.pb-c.c' failed

### DISCUSSION THREAD

**Discussion by rajgau11** [protobuf-c/protobuf-c#628] [Q&A] [answered]
Hello Sir

I recently got the protobuf-c git cloned:
git clone https://github.com/protobuf-c/protobuf-c.git
then performed:
./autogen.sh
./configure
make

While making it gives the following error:

vm11:~/protobuf-c$ make
  CXX      protoc-c/protoc_c_protoc_gen_c-c_bytes_field.o
  CXX      protoc-c/protoc_c_protoc_gen_c-c_enum.o
  CXX      protoc-c/protoc_c_protoc_gen_c-c_enum_field.o
  CXX      protoc-c/protoc_c_protoc_gen_c-c_extension.o
  CXX      protoc-c/protoc_c_protoc_gen_c-c_field.o
  CXX      protoc-c/protoc_c_protoc_gen_c-c_file.o
  CXX      protoc-c/protoc_c_protoc_gen_c-c_generator.o
  CXX      protoc-c/protoc_c_protoc_gen_c-c_helpers.o
  CXX      protoc-c/protoc_c_protoc_gen_c-c_message.o
  CXX      protoc-c/protoc_c_protoc_gen_c-c_message_field.o
  CXX      protoc-c/protoc_c_protoc_gen_c-c_primitive_field.o
  CXX      protoc-c/protoc_c_protoc_gen_c-c_service.o
  CXX      protoc-c/protoc_c_protoc_gen_c-c_string_field.o
  CXX      protoc-c/protoc_c_protoc_gen_c-main.o
  CXXLD    protoc-c/protoc-gen-c
  GEN      t/test.pb-c.c
/usr/local/bin/protoc: error while loading shared libraries: libprotoc.so.15: cannot open shared object file: No such file or directory
Makefile:2297: recipe for target 't/test.pb-c.c' failed
make: *** [t/test.pb-c.c] Error 127
vm11:~/protobuf-c$

Did I missed any step, please suggest?

Thanks
Rajeev

**Comment by wangkirin** [comment, marked as answer]
May be you  should specify the  `LD_LIBRARY_PATH`  in the Environmental Variable

For example:

In my machine,  the result of  `find / -name libprotoc.so`  is  `/usr/local/lib64/libprotoc.so`,  

so the command maybe 

```shell
export LD_LIBRARY_PATH=/usr/local/lib64
```
